airmozilla/new/templates/new/_email_new_event.html

```diff
@@ -19,6 +19,16 @@
   >{{ build_absolute_url(url('main:event', event.slug)) }}</a>
 </p>
 
+<p> 
+  {% if event.channels.all().exists() %}
+     Channels:
+     {% for channel in event.channels.all() %}
+          <a href="{{ build_absolute_url(url('main:home_channels', channel.slug)) }}">
+            {{ channel }}  </a> {% if not loop.last %} , {% endif %}
```

Comment:
There shouldn't be any white space between the `>` and the `<`. 
You can change it to be `... >{{ channel }}</a> ...`